plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.tsx

```diff
@@ -73,10 +73,10 @@ function readableEntityName(
   ref:
     | Entity
     | {
-        kind: string;
-        namespace?: string;
-        name: string;
-      },
+      kind: string;
+      namespace?: string;
+      name: string;
+    },
```

Comment:
Some formatting trouble here

```suggestion
        kind: string;
        namespace?: string;
        name: string;
      },
```

Reply:
Thanks, updated !